### PR TITLE
Added description how each user can change its own password

### PR DESCRIPTION
**Pull request by tonijurjevic96**
### Description

Added description how each user can change its own password

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

https://github.com/memgraph/memgraph/pull/2097

Closes:
(paste the link to the issue it closes)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

**Review by kgolubic — APPROVED** [review]
Everything looks OK. It can be merged after the development PR is merged.

File: pages/database-management/authentication-and-authorization/users.mdx
Comment (hal-eisen-MG) [inline pages/database-management/authentication-and-authorization/users.mdx:55]: Looks like the Grammarly check missed this

s/change own/change their own

Comment (kgolubic) [reply pages/database-management/authentication-and-authorization/users.mdx]: Thanks! Fixed.

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**
